Userland/Libraries/LibIPC/Message.cpp

```diff
@@ -75,8 +76,8 @@ ErrorOr<void> MessageBuffer::transfer_message(Core::LocalSocket& socket)
             if (auto error = maybe_nwritten.release_error(); error.is_errno()) {
                 // FIXME: This is a hacky way to at least not crash on large messages
                 // The limit of 100 writes is arbitrary, and there to prevent indefinite spinning on the EventLoop
-                if (error.code() == EAGAIN && writes_done < 100) {
-                    sched_yield();
+                if ((error.code() == EAGAIN || error.code() == EMSGSIZE) && writes_done < 100) {
```

Comment:
unix(7) doesn't document EMSGSIZE error. Do you have a reproducer for it?

Reply:
Yeah, it happens on yakos
```
[EMSGSIZE]         The socket requires that message be sent atomically, and the size of the message to be sent makes this impossible.  IOV_MAX.
```

Reply:
This _is_ present on linux as well.
> If the message is too long to pass atomically through the underlying protocol, the error EMSGSIZE is returned, and the message is not transmitted.

Reply:
at least macos returns EMSGSIZE because of buffer limitations, in this case it was doing that for a message of size 16.

Reply:
Maybe we should split the message up instead, but we might as well just drop the atomic message requirement (if we even have that) instad